actionpack/test/controller/request_forgery_protection_test.rb

```diff
@@ -529,11 +542,11 @@ def teardown
 
   def test_should_not_warn_if_form_authenticity_param_matches_form_authenticity_token
     ActionController::Base.logger = @logger
-    @controller.stubs(:valid_authenticity_token?).returns(:true)
-
     begin
-      post :index, params: { custom_token_name: 'foobar' }
-      assert_equal 0, @logger.logged(:warn).size
+      @controller.stub :valid_authenticity_token?, :true do
```

Comment:
@kaspth Take a look few lines up in the diff, I just moved this line inside `begin; end;` block.